server/admin_management/api/app/schemas.py

```diff
@@ -72,7 +72,7 @@ class FamUserBase(BaseModel):
 
 class FamUserDto(FamUserBase):
     user_type_code: famConstants.UserType
-    create_user: Annotated[str, StringConstraints(max_length=60)]
+    create_user: Annotated[str, StringConstraints(max_length=100)]
```

Comment:
this was left from previous tickets, just noticed them when doing test, the create/update user in database is having the limit 100, updated the schemas